play/PlayerRules/CharacterCreation/03_Feats.txt

```diff
@@ -457,76 +461,169 @@ Proficient: Explosives
+
+Steady Shot [5 Nanites]: 
+Prerequisites: PER 7
+Take 1 Action to breath slowly and line up your shot. -1 Ranged Miss Chance and 
++10m to the weapon's largest range bracket for each level of Steady Shot taken.
+Miss Chance can go below the -2 minimum range (See section "Combat:" in the basic
+rules document). You must be aiming at the target before you can use this feat. 
+Maximum 4 levels.
+
+Twitch	[7 Nanites]:
+Prerequisites: Level 3, Trained in SMGs or Pistols, DEX 7
 You may aim at an enemy as a free action instead of one action. Only 
-works with CQB weapons. +15% to miss with the aimed weapon.
+works with CQB weapons. +2 miss chance with the aimed weapon(s).
 
-Blur	[15 Nanites]:
+Blur	[11 Nanites]:
 Prerequisites: Level 10, Twitch
 You may aim at an enemy as a free action instead of one action. Only 
 works with CQB weapons. No Miss Chance penalty.
 
+====================
+Primary Stat Bonus Feats
+====================
+These feats cannot be bought. They are granted for free if you fulfill their prerequisites.
+
+Lucky: 
+Prerequisites: Luck 6
+	Good things predictably happen to you.
+	Luck = 6	
+		You can choose to reroll a die, once per long rest. You can only reroll 
+		once per separate die use and if you reroll, you must keep the second roll.
+	Luck = 7		
+		You can choose to reroll a die, twice per long rest. You can only reroll
+		once per separate die use and if you reroll, you must keep the second roll.
+	Luck = 8		
+		You can choose to reroll a die, twice per long rest. You can only reroll 
+		once per separate die use and if you reroll, choose which roll to keep.
+	Luck = 9		
+		You can choose to reroll a die, three times per long rest. You can only
+		reroll once per separate die use and if you reroll, choose which roll to keep.
+	Luck = 10		
+		You can choose to reroll a die, four times per long rest. You can reroll
+		as many times as you want (which uses up your rerolls-per-rest) and if you 
+		reroll, choose which of the rolls to keep.
+	Luck = 12	
+		You can choose to reroll a die with +10% or -10% bonus, four times per 
+		long rest. You can reroll as many times as you want (which uses up your 
+		rerolls-per-rest) and if you reroll, choose which of the rolls to keep.
+	Luck = 15	
+		You can choose to reroll a die with +20% or -20% bonus, four times per 
+		long rest. You can reroll as many times as you want (which uses up your
+		rerolls-per-rest) and if you reroll, choose which of the rolls to keep.
+		
+Positional Mastermind: 
+Prerequisites: Mutually Exclusive with Will to Survive; Charisma 9
+	You have incredible understanding of how beings in your universe perceive and react
+to their environments, allowing you to feint and position yourself in cover better. 
+If you have 9 Charisma, any living creature targeting you gets a +1 miss chance penalty
+when you have any kind of cover. If you have 12 Charisma, anything targeting you gets a 
++1 miss chance penalty when you have any kind of cover.

```

Comment:
no difference between 9 and 12 charisma?

Reply:
note to me: all caps "living creature" and "anything"